Remotion/Web/Development.WebTesting/Web.Development.WebTesting.csproj

```diff
@@ -22,10 +22,10 @@
     <ProjectReference Include="..\Contracts.DiagnosticMetadata\Web.Contracts.DiagnosticMetadata.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fork.Coypu" Version="3.3.0" />
+    <PackageReference Include="Fork.Coypu" Version="3.4.0-alpha.1" />
```

Comment:
we have a separate task for fork.coypu? if so, we can keep the alpha.1 here. otherwise, we need to update fork.coypu, too, to the next stable version.